packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -842,6 +863,12 @@ class ModalBottomSheetRoute<T> extends PopupRoute<T> {
   /// to have the bottom sheet be draggable.
   final bool isScrollControlled;
 
+  /// The max height constraint ratio for the bottom sheet
+  /// when [isScrollControlled] set to true.
+  ///
```

Comment:
Nit: maybe note that when [isScrollControlled] is set to false there is no constraint ratio.